### DISCUSSION TITLE
Create Table for MySQL

### DISCUSSION THREAD

**Discussion by AJW009** [PokeAPI/pokeapi#845] [Q&A]
I’m trying to find the CREATE TABLE statements so I can make my own copy in MySQL, I think Mariadb. Can anyone help me?

**Comment by Naramsim** [comment]
Hi, I don't think there's a `create table`. Everything is managed by Django. Your best bet is to clone the project, try to link it to your MySQL db and build the data. Probably you need to change the DB connector/engine. 

https://github.com/PokeAPI/pokeapi/blob/14c2f06bfb6fc440f5e2e73e2bc58cec48c85e7b/config/docker-compose.py#L7

**Reply by AJW009** [reply]
I’m a bit of a newbie to MySQL, do you mind me asking how I clone your database and change the db connector/engine?
